css/style.css

```diff
@@ -46,12 +46,14 @@ Developer @GA in NYC
 /*******************************************/
 
 body {
+  padding-top: 50px;
      color: #000;
      font-size: 12px;
      line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
+     /*font-family: Helvetica, Arial, sans-serif;*/
      background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
```

Comment:
:hammer: This background should be on just the top portion of the page, according to the pdf. Your HTML needs to be reorganized to let you style some of these things correctly.